recipes/flask-rest-orm/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "flask-rest-orm" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "e7a4ac59a70109687b0904eb36b81b04f6b2372ae013e280959d1e1d01f8af39" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ESSS/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - sqlalchemy >=1.1
+    - flask-restful >=0.3
+
+test:
+  imports:
+    - flask_rest_orm
+
+about:
+  home: https://github.com/ESSS/flask-rest-orm
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Flask extension to build REST APIs based on SQLAlchemy models'
+  description: |
+    A Flask extension to build REST APIs. It dismiss the need of building Schema classes, 
+    since usually all the information needed to serialize an SQLAlchemy instance is in the model itself.
+
+extra:
+  recipe-maintainers:
+    - igortg
+    - loliveira
```

Comment:
@lvoliveira and @edetec -- are you ok being added as co-maintainers of the `flask-rest-orm` recipe on conda-forge?

https://conda-forge.org/docs/recipe.html#maintainer-role

Reply:
yup, thanks

Reply:
yep, thanks